api/dss.yaml

```diff
@@ -1119,7 +1193,10 @@ components:
           anyOf:
           - $ref: '#/components/schemas/IdentificationServiceArea'
           description: Resulting service area stored in DSS.
-    GetIdentificationServiceAreasResponse:
+        version:
+          description: 'Version string required for replacing existing IdentificationServiceArea'
```

Comment:
This causes validation failure at SwaggerHub: "Sibling values are not allowed alongside $refs"  See https://github.com/OAI/OpenAPI-Specification/issues/556 for longer discussion

Reply:
removed description